src/main/resources/com/google/api/codegen/nodejs/version_index.snip

```diff
@@ -46,8 +46,10 @@
     {@comments(@service.doc.lines)}
     *
     * @@param {object=} options - [Configuration object](#/docs).
-    * @@param {number=} options.port - The port on which to connect to the remote host.
-    * @@param {string=} options.servicePath - The domain name of the API remote host.
+    * @@param {number=} options.port - The port on which to connect to
+    *   the remote host.
```

Comment:
We do 4 space indent -- can't remember if it is because of JSdocs/dox (the parser) having problems, or just convention. If it's not too annoying, feel free to change.